src/main/java/sirius/biz/storage/layer3/VirtualFile.java

```diff
@@ -1330,14 +1329,33 @@ public VirtualFile resolveOrLoadChildFromURL(URL url, boolean force, Predicate<S
             if (contentDispositionPath.isPresent() && fileExtensionVerifier.test(Files.getFileExtension(
                     contentDispositionPath.get()))) {
                 path = contentDispositionPath.get();
-            } else {
-                throw Exceptions.createHandled()
-                                .withNLSKey("VirtualFile.loadFromUrl.noValidPath")
-                                .set("url", url)
-                                .handle();
+            } else if (outcall.getResponseCode() == HttpResponseStatus.NOT_IMPLEMENTED.code()
+                       || outcall.getResponseCode() == HttpResponseStatus.SERVICE_UNAVAILABLE.code()) {
+                // some servers will respond with errors on head request, retry with a get request
+                outcall = new Outcall(url);
+                contentDispositionPath = outcall.parseFileNameFromContentDisposition();
+                lastModifiedHeader = outcall.getHeaderFieldDate(HttpHeaderNames.LAST_MODIFIED.toString());
+                // input has been opened by reading the header and as we don't read the body, close it to free resources
+                outcall.getInput().close();
```

Comment:
catch an ignore io exceptions here? we don't actually care if close comploetes with or without an exception we still want to continue